README.md

```diff
@@ -76,103 +88,156 @@ Of course, this Guide can’t replace professional advice. But talk to anyone wh
-  - Generally, it’s best to know the fully diluted number to know the likely percentage a number of shares is worth in the future. The terminology mentioned here isn’t universally applied, either, so it’s worth discussing it to be sure there is no miscommunication.
+- [**Public companies**](https://en.wikipedia.org/wiki/Public_company) are corporations where any member of the public can own stock. People can buy and sell the stock for cash on public [exchanges](https://www.investopedia.com/terms/e/exchange.asp). The value of a company’s shares is the value you see in the stock market reports, so shareholders know how much their stock is worth.
+- Most smaller companies, including all startups, are [**private companies**](https://en.wikipedia.org/wiki/Privately_held_company), where shareholders are restricted, meaning you can’t buy and sell stock easily or to anyone at any time.
+- ❗️ It’s hard to know precisely the value of private company stock. A stock certificate is a piece of paper that entitles you to something of highly uncertain value, and could well be worthless in the future, or highly valuable, depending on the fate of the company. Because the value of private company stock is not determined by public trades, shareholders make educated guesses about the likely future value, at a time when they will be able to buy or sell the stock.
+- Public and some private companies can pay [**dividends**](http://www.investopedia.com/terms/d/dividend.asp) to shareholders, but this is not common among startups and other companies focused on growth.
+- The ability to buy and sell stock is called **liquidity**. In startups and many private companies, it is typical to hold the stock until the company is sold or becomes public in an [**IPO**](https://en.wikipedia.org/wiki/Initial_public_offering). A sale or IPO is often called an **exit** or **liquidity event**. Sales, dissolutions, and bankruptcy are all called **liquidations**.
+- 🔹 **Private sales:** Liquidity events may also *never happen*-if, say, the company stays private or shuts down. But lacking the eventuality of a liquidity event, individuals may still buy or sell stock (“gain liquidity”) in private companies in specific ways:
+	- In a few cases, you may be able to sell private company stock to another private party, such as an accredited investor who wants to become an investor in the company. This is called a **secondary market** transaction. While many who hold stock hope or expect they need only find a willing buyer, in practice, secondary sales only work out in a few situations. In some cases, you may have luck selling stock privately to an individual, like a board member or former executive who wishes to increase their ownership. Quora has [more discussion](https://www.quora.com/How-do-employees-in-startups-sell-stock-in-the-secondary-markets) on this topic.
```

Comment:
Add a bit here about how companies frequently do secondary offerings in late stage funding rounds where employees are usually eligible to sell a % of their stock.

Reply:
✔️